content/workshops/pki_workshop/unit_lessons/unit_11.md

```diff
@@ -0,0 +1,80 @@
+# Unit 11: Certificate profiles
+
+In this unit you will go over the caUserCert profile. This certificate profile is for enrolling user certificates.
+
+The CA profiles are located here:
+
+    /var/lib/pki/ca1/ca/profiles/ca
+
+The [caUserCert.cfg](https://gitlab.consulting.redhat.com/pki/pki-workshop/blob/master/unit_labs/rhcs/caUserCert.cfg) profile is located in the rhcs labs folder.
+
+## Example caUserCert Profile
+
+The first part of a certificate profile is the description. This shows the name, long description, whether it is enabled, and who enabled it.
+
+    desc=This certificate profile is for enrolling user certificates.
+    visible=true
+    enable=true
+    enableBy=admin
+    name=Manual User Dual-Use Certificate Enrollment
+
+***
+
+The missing auth.instance_id= entry in this profile means that with this profile, authentication is not needed to submit the enrollment request.
+
+    auth.class_id=
+
+***
+
+Next, the profile lists all of the required inputs for the profile. For the **caUserCert** profile, this defines the keys to generate, the fields to use in the subject name, and the fields to use for the person submitting the certificate. 
```

Comment:
Next, in our case of the **caUserCert** profile, we view the required profile inputs. (input.list)
input.i1.class_id=keyGenInputImpl - defines the keys to generate
input.i2.class_id=subjectNameInputImpl - defines the keys to use in the subject name
input.i3.class_id=submitterInfoInputImpl - defines the fields to use for the person submitting the certificate